app/jobs/delete_pii_from_old_claims_job.rb

```diff
@@ -1,10 +0,0 @@
-# Runs weekly at 00:30 on a Sunday morning, and instructs the PiiScrubber class
-# to remove PII from eligible claims.
-class DeletePiiFromOldClaimsJob < CronJob
```

Comment:
Because we have renamed this scheduled job we'll need to manually unschedule it by removing the `Delayed::Job` record from the database to prevent the worker trying to load a class that doesn't exist and raising errors. You'll be able to identify it by examining the `handler` attribute on the job records in the database, it'll contain the a string like `"job_class: DeletePiiFromOldClaimsJob"` in it.

Reply:
Would you prefer that we do this in the console, or as a data migration in `db/data`?

Reply:
Ooh, good question! Maybe a good opportunity to experience writing a data migration?

Reply:
Added a data migration, could you take a look please?